srcpkgs/konquest/template

```diff
@@ -10,6 +10,6 @@ depends="hicolor-icon-theme"
 short_desc="KDE galactic conquest game"
 maintainer="Olga Ustuzhanina <me@laserbat.pw>"
 license="GPL-2.0-or-later"
-homepage="https://kde.org/applications/en/games/org.kde.konquest"
```

Comment:
They changed urls again and now both redirect to <https://apps.kde.org/konquest/>, please either leave as is or update to that.